src/components/BulkEditPane/BulkEditMarcLayer/BulkEditMarcLayer.js

```diff
@@ -86,7 +86,9 @@ export const BulkEditMarcLayer = ({
   });
 
   const hasBothFiles = bulkDetails?.linkToModifiedRecordsCsvFile && bulkDetails?.linkToModifiedRecordsMarcFile;
-  const areMarcAndCsvReady = hasBothFiles && isPreviewSettled;
+  const numberOfUnsupportedEntities = bulkDetails?.numberOfUnsupportedEntites || 0;
+  const numberOfSupportedEntities = bulkDetails?.processedNumOfRecords - numberOfUnsupportedEntities;
```

Comment:
I don't love that this calculation is repeated, though it is very simple. Maybe put in comments to cross reference the code so that if we find we need this in a _third_ place we can finally pull it out to a common function.